betScanner.py

```diff
@@ -0,0 +1,57 @@
+import time
+import logging
+from selenium import webdriver
+
+
+def main():
+    driver = webdriver.Chrome()
+    driver.get("https://www.marathonbet.ru/su/")
+    btn_tennis = driver.find_element_by_class_name("icon-sport-tennis")
+    btn_tennis.click()
+    start_time = time.time()
+    file_log = logging.FileHandler('Log.log')
+    console_out = logging.StreamHandler()
+    logging.basicConfig(handlers=(file_log, console_out), format='[%(asctime)s | %(levelname)s]: %(message)s',
+                        datefmt='%m.%d.%Y %H:%M:%S', level=logging.INFO)
+    matches = list(filter(delete_bets_without_odds, map(massive_of_matches, massive_of_table_strings(driver))))
+    print(matches)
+    logging.info('time for script execution %s seconds' % str(time.time() - start_time))
+    driver.close()
+    return len(matches)
+
+
+def delete_bets_without_odds(table_string):
```

Comment:
this function doesn't delete anything. should be renamed to something like _hasRates_. boolean function name should start from is, has, can...